readme.md

```diff
@@ -93,54 +96,65 @@ POST http://localhost:8000/notification/case/add
 ```
 
 ---
-#### Get Status
+
+## FCM Testing
+
+ #### GET /dashboard/status - Get the status of total case, death case, recovered case and suspect case by Covid-19
+
+
 ```
-GET http://localhost:8000/application/dashboard/status
+curl -H "Content-Type: application/json" -X GET http://localhost:8000/application/dashboard/status
 ```
 
-### FCM-Backend
+ #### If succeeded you should receive following JSON response with code `200`:
```

Comment:
this should be in API section. without curl :)

Reply:
fixed.